cosmovisor/.goreleaser.yml

```diff
@@ -0,0 +1,38 @@
+project_name: cosmovisor
+
+release:
+  disable: false
+  name_template: "{{.Tag}}"
+
+before:
+  hooks:
+    - go mod tidy
+
+builds:
+  - main: ./cmd/cosmovisor
+    goos:
+      - linux
+      - windows
+      - darwin
+    goarch:
+      - amd64
+      - arm64
+    env:
+      - CGO_ENABLED=0
+
+archives:
+  - name_template: '{{ .ProjectName }}-{{ replace .Version "cosmovisor/" "" }}-{{ .Os }}-{{ .Arch }}'
```

Comment:
why dropping "cosmovisor" from the release name?

Reply:
Where the `.ProjectName` and `.Version` is set? What are the values?

Reply:
> Where the .ProjectName and .Version is set? What are the values?

```
project_name: cosmovisor
```
and 
[`.Version`](https://github.com/goreleaser/goreleaser/blob/5a01a10f9bbc60ab75aff0b2971af75d32436bdf/internal/pipe/git/git.go#L37) is `GORELEASER_CURRENT_TAG` but trimmed  with prefix `v`. In this case, no trim so should be same as `.Tag`

> why dropping "cosmovisor" from the release name?

In archives `name_template`,  `cosmovisor/v0.1.0` will be converted to `cosmovisor.v0.1.0` .
And, `cosmovisor-v0.1.0` looks better?

Reply:
so, it's simpler to do `{{ replace .Version "cosmovisor/" "comsovisor-" }}-{{ .Os }}-{{ .Arch }}'` without doing other statements / variables.